Big-Spender/readme.md

```diff
@@ -138,3 +157,33 @@ INSERT YOUR QUERIES HERE
 - [ ] All queries are written in SQL
 - [ ] All queries are correct and I have tested them in the database
 - [ ] I have opened a pull request with my answers written directly into this README.md file
+
+
```

Comment:
This is slightly confusing to have all of them right here! It would probably be helpful if you could have this formatted a bit better :)